viewer/packages/pointclouds/src/potree-three-loader/Potree.ts

```diff
@@ -111,8 +111,8 @@ export class Potree implements IPotree {
     ray: Ray,
     params: Partial<PickParams> = {}
   ): PickPoint | null {
-    Potree.picker = Potree.picker || new PointCloudOctreePicker();
-    return Potree.picker.pick(renderer, camera, ray, pointClouds, params);
+    Potree.picker = Potree.picker || new PointCloudOctreePicker(renderer);
```

Comment:
Hmmmmmmmm! This looks scary, you commit to using the same renderer for all picking. What happens if you instantiate two viewers with different renderers and do point cloud picking?

Reply:
Is `Potree.picker` a static global? If yes, is it really necessary?

Reply:
Removed it